artemis-serialization/artemis-json-libgdx/src/main/java/com/artemis/io/JsonArtemisSerializer.java

```diff
@@ -63,7 +63,7 @@ public JsonArtemisSerializer setUsePrototypes(boolean usePrototypes) {
 		return this;
 	}
 
-	public void save(Writer writer, SaveFileFormat save) {
+	private void save(Writer writer, SaveFileFormat save) {
```

Comment:
Save should be public

Reply:
But its only usage is inside protected method `save(OutputStream, SaveFileFormat)`. Why do you see it as a part of public API? Do you think anybody use in some way?
Btw same question regarding `JsonArtemisSerializer#save(Writer, SaveFileFormat)` which is part of `artemis-json-libgdx` module.

Reply:
I seem to remember that we had to use it in a previous project, but my memory maybe betrays me as `WorldSerializationManager::save` is indeed sufficient. In any case, i'd rather wait with reducing visilibty of the method until when we break the API and bump the version to 3.0.0.

Maybe just mark the *Serializer::save(Writer, SaveFileFormat) methods as deprecated instead, alternatively just leave them be until 3.0.0 is upon us.